charts/consul/test/terraform/eks/main.tf

```diff
@@ -61,7 +68,7 @@ module "eks" {
   kubeconfig_api_version = "client.authentication.k8s.io/v1beta1"
 
   cluster_name    = "consul-k8s-${random_id.suffix[count.index].dec}"
-  cluster_version = "1.23"
+  cluster_version = "1.26"
```

Comment:
Same comment as above about the version.

Reply:
Talked to @thisisnotashwin I'll pin these to a patch version as well for better repro-ability